### PR TITLE
Add AWS Systems Manager Plugin to  pre-requisite in README

### PR DESCRIPTION
**Pull request by jasonodoom**
## 🗣 Description ##

This pull request adds the installation of AWS Systems Manager as a requirement to Pre-requisites.

## 💭 Motivation and context ##

This was missing during the configuration of awssh and @dav3r recommended I open a PR.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ This project provides a tool that simplifies secure shell connections over
 ## Pre-requisites ##
 
 - The [AWS CLI](https://aws.amazon.com/cli/) installed on your system.
+- The [AWS SM plugin](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html) installed on your system.
```
Comment (mcdonnnj) [inline README.md]: This should also resolve your linting failure.

```suggestion
- The [Session Manager plugin for the AWS CLI](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html)
  installed on your system.
```

**Review by mcdonnnj — APPROVED** [review]
Pretty straightforward. I have one suggestion for your consideration.

**Review by jsf9k — CHANGES_REQUESTED** [review]
@jasonodoom - I'll approve once you add @mcdonnnj's suggestion.  I think "SM" needs to be written out here for clarity, so I'm making it a required change.

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
👍

**Pull request merged**